src/types.ts

```diff
@@ -135,6 +135,7 @@ export type NavigationState = {
     amount?: string;
     customRecords?: Record<string, string>;
     message?: string;
+    event?: Event;
```

Comment:
I added this to this type, but IMHO that feels very quite wrong as this type has to hold ALL types of all calls. Don't know if we want to tackle that within this PR though...

Reply:
cc @escapedcat @lisabaut